config/shenzen_neo/nas-cs01z.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product xmlns='https://github.com/OpenZWave/open-zwave' Revision="1">
+    <!-- NAS-CS01Z
```

Comment:
Please add MetaData Block
https://github.com/OpenZWave/open-zwave/wiki/Adding-Devices#creating-the-metadata-block

Reply:
Hi, trying to build the metadata file, but I cannot find the device on the Z-Wave alliance website. However I found a complete description of it here: https://www.cd-jackson.com/index.php/zwave/zwave-device-database/zwave-device-list/devicesummary/1108

Reply:
`<MetaData>
	<MetaDataItem name="Name">NEO Coolcam Roller Shutter</MetaDataItem>
	<MetaDataItem name="Description">NEO Coolcam Roller Shutter - Curtain Switch - NAS-SC03ZE-2-T-V3</MetaDataItem>
	<MetaDataItem name="OzwInfoPage">http://www.openzwave.com/device-database/0258:102c:0200</MetaDataItem>
	<MetaDataItem name="ProductPage">https://szneo.com/en/products/show.php?id=234</MetaDataItem>
	<MetaDataItem name="ProductSupport">https://szneo.com/en/service/index.php</MetaDataItem>
	<MetaDataItem name="ProductPic">https://www.cd-jackson.com/zwave_device_uploads/1108/neo-curtain-Switch.JPG</MetaDataItem>
	<MetaDataItem name="ProductManual">https://www.cd-jackson.com/zwave_device_uploads/1108/NeoRollerS.pdf</MetaDataItem>
	<MetaDataItem name="WakeupDescription">Power on the device</MetaDataItem>
	<MetaDataItem name="InclusionDescription">1. Set Zwave controller intro inclusion mode
2. power on the device
3. touch the middle button 3 times within 1s
4. the device will be entered into inclusion mode automatically
(All Led lights will be bliked with 1s interval until inclusion successfull)</MetaDataItem>
	<MetaDataItem name="ExclusionDescription">1. Set Z-WaveControll into exclusion mode
2. Power on the device
3. Touch the middle button 3 times within 1s
4. the device will be entered into exclusion mode
(All led lights will de blinked 3 times with 0.5s interval.)</MetaDataItem>
	<MetaDataItem name="ResetDescription">Power on the device
	Keep touching the middle button for 10s until the led turns to red, then release</MetaDataItem>
	<MetaDataItem id="102C" name="ZWProductPage" type="0200">Unknown</MetaDataItem>
	<MetaDataItem id="102C" name="FrequencyName" type="0200">CEPT (Europe)</MetaDataItem>
	<MetaDataItem id="102C" name="Identifier" type="0200">NAS-SC03ZE-2-T-V3</MetaDataItem>
	<ChangeLog>
		<Entry author="John Huppertz - jhuppertz@ziggo.nl" date="02 December 2020" revision="1">Initial setup of metadata file</Entry>
	</ChangeLog>
</MetaData>`